Explorer/Assets/DCL/Utilities/EnsUtils.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Text.RegularExpressions;
+
+namespace DCL.Utilities
+{
+    public static class EnsUtils
+    {
+        private const string WORLD_URL = "https://worlds-content-server.decentraland.org/world/";
+
+        public static string ConvertEnsToWorldUrl(string ens) =>
+            WORLD_URL + ens.ToLower();
+
+        public static bool ValidateEns(string ens) =>
```

Comment:
I recommend introducing a special wrapper type to indicate `ENS` instead of generic string just like it's done for `URN`

Reply:
good point! done!